src/common/commandExecutor.ts

```diff
@@ -125,9 +125,15 @@ export class CommandExecutor {
            Log.logStreamData(data, process.stdout);
         });
 
+        // HACK: Remove this and make a better implementation
```

Comment:
don't check this in with hack in the comment, create an issue and reference it here